### ISSUE TITLE
Save states and Hotkeys

### ISSUE THREAD

**Issue by bajolzas** [byuu/bsnes#107]
First of all sorry for all these recent requests, as bsnes seems to lack stuff to ask about this may be my last request... in the near future.

Anyway, in bsnes I currently have F1 as Save state, and F2 as Load state, but as I am not used to this configuration sometimes I end up saving when I want to load, or load when I want to save.

This is because I am used, on other emulators, to have F1(F2, F3...) to load states and LShift+F1(F2, F3...) to save states.

In the Hotkeys tab I've tried setting Combinational logic to "And" and having Save state Mapping #1 set to Lshift and Mapping #2 set to F1, and it works; but if I have F1 set to Load state, it will allways load no matter if I press LShift or not.

This Combinational logic set to "And" and not "Or" also has the adverse effect of me having to sacrifice having a given hotkey (in my case fast forward) set to both my keyboard and my controller.

Both Desmume and snes9x only accept the hotkey input change when the key is released wich lets me input both LShift and F1, unlike bsnes that accepts just LShift right away.

So long story short, I want to be able to load state with F1 and save state with LShift+F1 (if possible without sacrificing having multiple inputs in other hotkeys).

If we could have an input pair for every save slot then better.

Thanks

**Comment by byuu** [comment]
I understand the issue but I'm not sure I can fix this one with the way my input mapping system works. I would have to special-case the alt/control/super keys as modifiers, which would be a pretty significant change to any existing users that map them to action keys.

**Comment by byuu** [comment]
Closing as I can't really fix this, going to have to say no to the request for now, sorry.
